README.md

```diff
@@ -63,6 +63,18 @@ repository.
 If you encounter any issues whilst using cert-manager, and your issue is not
 documented, please [file an issue](https://github.com/jetstack/cert-manager/issues).
 
+Possible resources for *help*:
+
+* The official channel `#cert-manager` on `kubernetes.slack.com`
```

Comment:
A link to the signup page would also help - I remember it not being immediately obvious where to go to get an invite: http://slack.kubernetes.io/